### PR TITLE
Update styles and bower.

### PR DESCRIPTION
**Pull request by cdata**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -8,9 +12,9 @@
   "devDependencies": {
     "test-fixture": "polymerelements/test-fixture#^0.8.0",
     "iron-iconset": "polymerelements/iron-iconset#^0.8.0",
-    "core-icons": "polymerelements/iron-icons#^0.8.0",
+    "iron-icons": "polymerelements/iron-icons#^0.8.0",
     "iron-doc-viewer": "polymerelements/iron-doc-viewer#^0.8.0",
     "webcomponentsjs": "webcomponents/webcomponentsjs#^0.6.0",
-    "web-component-tester": "~2.2.6"
+    "web-component-tester": "*"
   }
```
Comment (notwaldorf) [inline bower.json:19]: No more ^ style deps for web-component-tester?

**Comment by notwaldorf** [comment]
Your tests have poetic errors ( "expected undefined to deeply equal '0px'"). Other than that, lgtm and a question :)

File: bower.json (hunk 1 above)
Comment (cdata) [inline bower.json:19]: @nevir can clarify if there is a better choice, but last time he told me that both `^` and `*` are acceptable, just not `~`.

**Pull request merged**